F21/cemilcan98/app.py

```diff
@@ -0,0 +1,84 @@
+def toLowerCase(i):
+    if(i[1] in ["en", "en-US", "en-IE", "en-Latn"]):
+        y = i[0].lower()
+        return y
+
+    # Checking for the special letters in Turkish and Azeri 
+    if(i[1] == "tr" or i[1] == "az"):
+        y = i[0]
+        y = y.replace('\u011E', '\u011F')

```

Comment:
The Unicode code points are not very readable here; either use the explicit characters or explain what these are in comments.